packages/server/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * The service-worker broadcast channel id
+ */
+export const SERVICE_WORKER_BROADCAST_CHANNEL_ID = '/sw-api.v1';
```

Comment:
Do we need a new file to export this constant? Looks like `tokens.ts` already exports `WORKER_NAME`, so maybe we could also put this const there?

Reply:
I wanted a safe place for the service-worker to import only simple stuff without pulling the entire world. Turns out the service-worker cannot import from that file without breaking completely, so I'll just get rid of it.